.ci/pytorch/test.sh

```diff
@@ -342,6 +342,7 @@ test_dynamo_wrapped_shard() {
     exit 1
   fi
   python tools/dynamo/verify_dynamo.py
+  python tools/dynamo/gb_id_mapping.py verify
```

Comment:
This place should technically work, although it's not the ideal place since dynamo_wrapped is the shard where we run most PyTorch unittests, but wrapped with torch.compile. The default shard might be a better place to place this, but I'm not sure where that would be in this script.

This comment does not need to block landing